src/test/java/org/opensearch/security/dlic/dlsfls/DlsTest.java

```diff
@@ -44,6 +51,7 @@ protected void populateData(Client tc) {
         System.out.println("q");
         System.out.println(Strings.toString(XContentType.JSON, tc.search(new SearchRequest().indices(".opendistro_security")).actionGet()));
```

Comment:
out of scope of this PR; but we can get rid of these sysouts to boost test performance.